sql/src/main/java/io/crate/analyze/CreateAnalyzerStatementAnalyzer.java

```diff
@@ -258,4 +261,18 @@ private String extractType(GenericProperties properties, ParameterContext parame
         return ExpressionToStringVisitor.convert(expression, parameterContext.parameters());
     }
 
+    private static void validateCharFilterProperties(String type, @Nullable GenericProperties properties) {
+        if (type.equals("mapping")) {
```

Comment:
I would prefer the opposite (only allow html_strip instead of forbidding the others), so if new charFilter would be implemented in future, it won't break